hermes/file_gen.go

```diff
@@ -1,28 +1,28 @@
-package main
+package hermes
+
 import (
 	"fmt"
-	// "math/rand"
 	"strconv"
 	"crypto/sha1"
 )
 type HermesFile struct {
 	name string
 	contents string
 }
-func generate_file_name (file_ID int, file_checksum string) string{
-	var id_string string=""
+func (file *HermesFile) GenerateFileName(file_ID int, file_checksum string) {
+	id_string :=""
 	if file_ID < 10 {
 		id_string = "0" + strconv.Itoa(file_ID)
 	} else {
 		id_string = strconv.Itoa(file_ID)
 	}
-	var file_name string = "Hermes_"+id_string+file_checksum
-	return file_name 
+	file_name := "Hermes_"+id_string+file_checksum
+	file.name = file_name
 }
-func generate_file_contents (file_ID int) string{
-	return "jhfvjhdfjhfjjhjhdfvjvcvfjh";
+func (file *HermesFile) GenerateFileContents(file_ID int) {
```

Comment:
- nit: leave a blank line between functions.
- Generating file contents also shouldn't be public.
```suggestion

func (file *HermesFile) generateFileContents(file_ID int) {
```